core/execution/future/market.go

```diff
@@ -1417,7 +1403,7 @@ func (m *Market) leaveAuction(ctx context.Context, now time.Time) {
 
 	m.checkForReferenceMoves(ctx, updatedOrders, true)
 
-	m.checkLiquidity(ctx, nil, true)
+	m.checkLiquidity(ctx)
```

Comment:
could just call here m.checkBondBalance(ctx) I suppose and get rid of checkliquidity?